packages/expo-ui/src/ExpoUIView.tsx

```diff
@@ -1,10 +1,18 @@
 import { requireNativeView } from 'expo';
 import * as React from 'react';
 
-import { ExpoUIViewProps } from './ExpoUI.types';
+import { ButtonProps, SingleChoiceSegmentedControlProps } from './ExpoUI.types';
 
-const NativeView: React.ComponentType<ExpoUIViewProps> = requireNativeView('ExpoUI');
+const SingleChoiceSegmentedControlViewNativeView: React.ComponentType<SingleChoiceSegmentedControlProps> =
+  requireNativeView('ExpoUI', 'SingleChoiceSegmentedControlView');
 
-export default function ExpoUIView(props: ExpoUIViewProps) {
-  return <NativeView {...props} />;
+const ButtonNativeView: React.ComponentType<ButtonProps> = requireNativeView('ExpoUI', 'Button');
+
+export function SingleChoiceSegmentedControlView(props: SingleChoiceSegmentedControlProps) {
+  return <SingleChoiceSegmentedControlViewNativeView {...props} />;
+}
+
+export function Button(props: ButtonProps) {
+  // Min height from https://m3.material.io/components/buttons/specs, minWidth
+  return <ButtonNativeView style={[{ minWidth: 80, minHeight: 40 }, props.style]} {...props} />;
```

Comment:
Will setting an explicit width/height prevent the underlying native views from adjusting the height contextually based on usage. For example, in SwiftUI, using a `Button` in a `List` changes the size to fill the list, and using it in a toolbar (headerLeft in RNScreens) should change it to be more optimized for the smaller space.

Reply:
We're pretty limited in what we can do with SwiftUI – in particular we can't drive parent size based on children size yet